CarpetLib/src/restrict_3d_stagger111.cc

```diff
@@ -137,8 +137,8 @@ void restrict_3d_stagger111(
 
   } else {
 
-// Loop over coarse region
-#pragma omp parallel
+    // Loop over coarse region
+    // #pragma omp parallel
```

Comment:
this should not be committed in this form. Rather the whole pragma should be removed.